### PR TITLE
Add responsive behaviour

### PR DESCRIPTION
**Pull request by oliverturner**
Add responsive behaviour and remove alert on window width

Preview here: https://5ca3593539bdd6fb07a3dd9a--perflink.netlify.com

Screenshot 2019-04-02 at 13 48 27

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
+    <meta charset="utf-8"/>
+    <script>
+      sessionStorage.redirect = location.href;
+    </script>
+    <meta http-equiv="refresh" content="0;URL='/'">
+    <link rel="stylesheet" href="/index.css">
+  </head>
 
-<body>
-  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (lukejacksonn) [inline 404.html]: This is required to make GitHub pages recognise it as a real 404.html (has to be over 512 bytes). Might not still be a thing.. I haven't checked recently.

File: index.css [hunk 2]
```diff
@@ -36,3 +35,59 @@ button:hover {
...
+  color: rgba(255, 255, 255, 0.62);
+  max-height: 100vh;
+  overflow-y: auto;
+}
+
+.graph {
+  grid-area: graph;
+
+  display: flex;
+  justify-content: space-around;
```
Comment (lukejacksonn) [inline index.css]: I had space-around set at first but I think I prefer center, also needs some padding.

File: routes/home/index.js [hunk 3]
```diff
@@ -18,8 +18,8 @@ const init = location.hash
       tests: [
         { code: '' },
         { code: 'data.find(x => x == 5000)' },
-        { code: 'data.find(x => x == 10000)' },
-      ],
+        { code: 'data.find(x => x == 10000)' }
+      ]
```
Comment (lukejacksonn) [inline routes/home/index.js]: Enable trailing commas to reduce diff maybe? You could add a prettier.rc I guess; no semis and trailing commas ;)

File: index.css [hunk 4]
```diff
@@ -36,3 +35,59 @@ button:hover {
...
+  color: white;
+}
+
+.missing a {
+  color: #b6b6b6;
+  margin-top: 3rem;
+}
+
+.app {
+  display: grid;
```
Comment (lukejacksonn) [inline index.css]: Loving the use of grid here ❤️

File: routes/home/index.js [hunk 5]
```diff
@@ -100,14 +100,8 @@ export default () => {
             </p>
             <button
               onClick=${_ => {
-                if (window.innerWidth < 900) {
```
Comment (lukejacksonn) [inline routes/home/index.js]: I'd still like to address how the _control panel_ within each test case looks on really small screens (like mobile small)

Comment (lukejacksonn) [reply routes/home/index.js]: Currently this happens:
Screen Shot 2019-04-02 at 17 44 10

File: .prettierrc [hunk 6]
```diff
@@ -0,0 +1 @@
+{ "semi": false, "singleQuote": true }
```
Comment (lukejacksonn) [inline .prettierrc]: Ahh here is the rc. Just add trailing ES5 trailing commas and I'm happy 👍

**Comment by lukejacksonn** [comment]
@oliverturner thanks a bunch for taking the time to address this very messy CSS 😅 I have left some comments. Feel free to address them otherwise I will get around to it when I can!

File: .prettierrc (hunk 6 above)
Comment (oliverturner) [reply .prettierrc]: Done and done

File: routes/home/index.js (hunk 5 above)
Comment (oliverturner) [reply routes/home/index.js]: localhost_8080_(iPhone 6_7_8 Plus)

Comment (oliverturner) [reply routes/home/index.js]: iPhone 5:
localhost_8080_(iPhone 5_SE)

iPhone 6/7/8 Plus:
localhost_8080_(iPhone 6_7_8 Plus)

File: 404.html (hunk 1 above)
Comment (oliverturner) [reply 404.html]: Maybe add an encouraging bon mot instead? 

"No codez for you lol 😂" or similar?

File: index.css (hunk 2 above)
Comment (oliverturner) [reply index.css]: Fair play 👍

**Comment by oliverturner** [comment]
Good feedback. Addressed most points 🙌

File: routes/home/index.js (hunk 5 above)
Comment (oliverturner) [reply routes/home/index.js]: Check the live link below for update

**Comment by lukejacksonn** [comment]
I pulled this down last night and have made some final tweaks (a single commit on top of yours). Are you ok with me pushing them straight to master when I'm done?

**Comment by oliverturner** [comment]
Of course! 😀

**Comment by lukejacksonn** [comment]
Merged in these changes https://github.com/lukejacksonn/perflink/pull/14 🎉 thank you @oliverturner for making this project more awesome!

**Pull request closed without merging**